core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -68,6 +69,8 @@ export default {
     PartnershipsFormInPortuguese:
       'https://docs-google-com.translate.goog/forms/d/e/1FAIpQLSdL5mjFO7RxDtg8yfXluEtciYj8WnAqTL9fZWnwPgOqXV-9lg/viewform?_x_tr_sl=en&_x_tr_tl=pt&_x_tr_hl=en-US&_x_tr_pto=wapp',
     PartnershipsFormShortUrl: 'https://forms.gle/Y71U8FdhQwZpicJj8',
+    releaseCoordinatorPage: 'http://localhost:8181/release-coordinator',
```

Comment:
'ReleaseCoordinator"

Reply:
Done.

Reply:
It's still not changed?

Reply:
ah, missed.  Updated though. 

Thanks.